### ISSUE TITLE
Saving Metadata to the MySQL

### ISSUE THREAD

**Issue by IceFlovver** [amosproj/amos2022ws01-firmware-scraper#16]
## User story
1. As a user after having found the firmware on vendor's website
2. I want the software to download the metadata and save it to MySQL database
3. So that it can be stored in the database

## Acceptance criteria
* it is possible to give an input string to the program
* data is saved in the database
* the program returns successful execution or error message

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
